common/src/androidTest/java/com/microsoft/identity/common/CommandDispatcherTest.java

```diff
@@ -478,6 +506,26 @@ public boolean isEligibleForEstsTelemetry() {
         }
     }
 
+    static class IErrorExceptionCommand extends BaseCommand<String> {
```

Comment:
Does not seem an interface 

```suggestion
    static class ErrorExceptionCommand extends BaseCommand<String> {
```

Reply:
I've renamed it.